server/settings/main.py

```diff
@@ -16,6 +17,12 @@ class ShotgridServiceSettings(BaseSettingsModel):
         validate_default=False,
     )
 
+    ayon_service_user: str = Field(
+        default="service",
+        title="The AYON Shotgird user",
```

Comment:
How about
```suggestion
        title="AYON service user",
```
?
since it's not really SG specific?

Reply:
Hmm, you could have multiple service users, and in here we need to pick the one that we use when setting up the services/ash through the `AYON_API_KEY` ; which after writing this maybe could lead to mismatch of things, say we use another service (unrelated to shotgrid) to create an entity cause they share service name, the transmitter would ignore it eve tho it shouldn't so, maybe I should enforce (or create) a user that matches the secret username for SG? Thoughts?

Reply:
yeah I'd say so, if it can happen, it will happen haha